hack/lib/protoc.sh

```diff
@@ -26,16 +26,33 @@ KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
 PROTOC_VERSION=23.4
+PROTOC_GEN_GO_VERSION=1.35.1
+PROTOC_GEN_GO_GRPC_VERSION=1.5.1
 
 # Generates $1/api.pb.go from the protobuf file $1/api.proto
 # and formats it correctly
 # $1: Full path to the directory where the api.proto file is
-function kube::protoc::generate_proto() {
+function kube::protoc::generate_proto_gogo() {
   kube::golang::setup_env
   GOPROXY=off go install k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
 
   kube::protoc::check_protoc
 
+  local package=${1}
+  kube::protoc::protoc_gogo "${package}"
+  kube::protoc::format "${package}"
+}
+
+# Generates $1/api.pb.go from the protobuf file $1/api.proto
+# and formats it correctly without using gogo
+# $1: Full path to the directory where the api.proto file is
+function kube::protoc::generate_proto() {
+  kube::golang::setup_env
+  go install "google.golang.org/protobuf/cmd/protoc-gen-go@v$PROTOC_GEN_GO_VERSION"
+  go install "google.golang.org/grpc/cmd/protoc-gen-go-grpc@v$PROTOC_GEN_GO_GRPC_VERSION"
```

Comment:
should we manage these versions via bash constants, go.mod requires in our main module, or go.mod requires in our hack/tools module?